logger/module.go

```diff
@@ -57,8 +57,13 @@ func Create(bus modules.Bus, options ...modules.ModuleOption) (modules.Module, e
 	return new(loggerModule).Create(bus, options...)
 }
 
-func (*loggerModule) CreateLoggerForModule(moduleName string) modules.Logger {
-	return Global.Logger.With().Str("module", moduleName).Logger()
+// CreateLoggerForModule returns the logger with additional context (e.g. module name)
```

Comment:
Not actionable.

Thanks for this. We definitely don't do a good job at using godoc at the moment, so do continue doing this whenever you find an opportunity.